### ISSUE TITLE
Changing handles colour in text-input (Desktop)

### ISSUE THREAD

**Issue by capecrusader-121** [kivy/kivy#8373]
**Is your feature request related to a problem? Please describe.**
In kivy widget  of text-input , the use handles option in desktop doesn't have a property i could find to change its colour, it default to light blue, i want to change its colour to other elements of my app .

**Describe the solution you'd like**
A quick & easy way to modify its colour , through colour property or something similar. 

**Describe alternatives you've considered**
I tried checking the source code to get some hints on changing it but i couldn't find one.

**Additional context**
![screen](https://github.com/kivy/kivy/assets/100503303/093964e8-6864-4486-91df-9d42abd588da)

**Comment by p0lygun** [comment]
so u want to change the color of the text that is in the text input widget?

**Comment by p0lygun** [comment]
ohh the pop up thingy

**Comment by p0lygun** [comment]
Kivy does provide a way to override the said handles, You need to [override](https://github.com/kivy/kivy/blob/ca1b918c656f23e401707388f25f4a63d9b8ae7d/kivy/uix/textinput.py#L3876) the `handle_image_left`, `handle_image_middle`, `handle_image_right`  according to your needs.

**Comment by capecrusader-121** [comment]
@p0lygun  Thnks , i can see that it needs one creating custom atlas pics for tis nevertheless wont it would have been good if one could use something like colour property to change the colour ?

**Comment by p0lygun** [comment]
It is possible but you need to make a little function, here is something I came up with, 
```py
def set_handles_color(input_instance: TextInput, color):
    # self._handle_left = handle_left = Selector()
    win = EventLoop.window
    for handle_name in ['left', 'right', 'middle']:
        handle = Selector(
            source=getattr(input_instance, f"handle_image_{handle_name}"),
            target=input_instance,
            window=win,
            size_hint=(None, None),
            size=("45dp", "45dp"),
        )
        handle.color = color
        handle.bind(
            on_press=input_instance._handle_pressed,
            on_touch_move=input_instance._handle_move,
            on_release=input_instance._handle_released,
        )
        setattr(input_instance, f"_handle_{handle_name}", handle)
```
you can use this as such
```py
from kivy.uix.textinput import TextInput, Selector
from kivy.base import runTouchApp, EventLoop


def set_handles_color(input_instance: TextInput, color):
    win = EventLoop.window
    for handle_name in ['left', 'right', 'middle']:
        handle = Selector(
            source=getattr(input_instance, f"handle_image_{handle_name}"),
            target=input_instance,
            window=win,
            size_hint=(None, None),
            size=("45dp", "45dp"),
        )
        handle.color = color
        handle.bind(
            on_press=input_instance._handle_pressed,
            on_touch_move=input_instance._handle_move,
            on_release=input_instance._handle_released,
        )
        setattr(input_instance, f"_handle_{handle_name}", handle)


if __name__ == "__main__":
    text_input_with_custom_handles = TextInput(
        size_hint=(.5, .5),
        use_handles=True,
        pos_hint={'center_x': .5, 'center_y': .5},
    )
    set_handles_color(text_input_with_custom_handles, [1, 0, 0, 1])
    runTouchApp(text_input_with_custom_handles)
```

**Comment by p0lygun** [comment]
I agree with adding a property tho. +1

**Comment by akshayaurora** [comment]
https://github.com/kivy/kivy/blob/master/kivy/uix/textinput.py#L240

So in your code you can just extend the template of `Selector`

```
<Selector>
    color: 0,1,0, 1
```

I guess we just need to document it .

**Comment by capecrusader-121** [comment]
@akshayaurora  How do one implement the stated method in a kivy file ? I mean i have defined all widgets within the kivy file itself and using the python file only for threads and logic.

**Comment by p0lygun** [comment]
> @akshayaurora  How do one implement the stated method in a kivy file ? I mean i have defined all widgets within the kivy file itself and using the python file only for threads and logic. 

Just put the same in the kivy file that is loaded first

**Comment by p0lygun** [comment]
@akshayaurora i think we can close this now

**Comment by akshayaurora** [comment]
Should be fixed by #8377
